.golangci.yml

```diff
@@ -310,7 +308,7 @@ linters-settings:
       # https://github.com/mgechev/revive/blob/master/RULES_DESCRIPTIONS.md#function-length
       - name: function-length
         severity: warning
-        disabled: false
+        disabled: true
```

Comment:
is it relevant to the PR?

Reply:
I think this is by the change of latest develop cause I didn't change this file and just merged latest